tests/hstestcase.py

```diff
@@ -9,10 +11,11 @@ class HSTestCase(unittest.TestCase):
     projectid = str(random.randint(2222000, 2223000))
     spidername = 'hs-test-spider'
     endpoint = os.getenv('HS_ENDPOINT', 'http://storage.vm.scrapinghub.com')
+    jobq_endpoint = os.getenv('JOBQ_ENDPOINT', 'http://jobq.vm.scrapinghub.com:5002')
```

Comment:
I created this `job_endpoint` because I had no success accessing `/botgroups` endpoint through `storage.vm.scrapinghub.com`. Is there any other "partial" that should be used after/before it (besides `/jobq`)?